Description: Fix Ruby 2.7 warning
 The '**' operater should be used to pass keyword arguments from now on.
Author: Lucas Kanashiro <lucas.kanashiro@canonical.com>
Last-Updated: 26-03-2020
Forwarded: yes, https://github.com/puppetlabs/puppet/pull/8060

--- a/lib/puppet/file_system/file_impl.rb
+++ b/lib/puppet/file_system/file_impl.rb
@@ -77,7 +77,7 @@
   end
 
   def read(path, opts = {})
-    path.read(opts)
+    path.read(**opts)
   end
 
   def read_preserve_line_endings(path)
